dlp/src/main/java/com/example/dlp/DeId.java

```diff
@@ -0,0 +1,237 @@
+          ContentItem.newBuilder()
+              .setType("text/plain")
+              .setValue(string)
+              .build();
+
+      CharacterMaskConfig characterMaskConfig =
+          CharacterMaskConfig.newBuilder()
+              .setMaskingCharacter(maskingCharacter.toString())
+              .setNumberToMask(numberToMask)
+              .build();
+
+      PrimitiveTransformation primitiveTransformation =
+          PrimitiveTransformation.newBuilder()
+              .setCharacterMaskConfig(characterMaskConfig)
+              .build();
+
+      InfoTypeTransformation infoTypeTransformationObject =
+          InfoTypeTransformation.newBuilder()
+              .setPrimitiveTransformation(primitiveTransformation)
+              .build();
+
+      InfoTypeTransformations infoTypeTransformationArray =
+          InfoTypeTransformations.newBuilder()
+              .addTransformations(infoTypeTransformationObject)
+              .build();
+
+      DeidentifyConfig deidentifyConfig =
+          DeidentifyConfig.newBuilder()
+              .setInfoTypeTransformations(infoTypeTransformationArray)
+              .build();
+
+      DeidentifyContentRequest request =
+          DeidentifyContentRequest.newBuilder()
+              .setDeidentifyConfig(deidentifyConfig)
+              .addItems(contentItem)
+              .build();
+
+      DeidentifyContentResponse response = dlpServiceClient.deidentifyContent(request);
+
+      for (ContentItem item : response.getItemsList()) {
+        System.out.println(item.getValue());
+      }
+    } catch (Exception e) {
+      System.out.println("Error in deidentifyWithMask: " + e.getMessage());
+    }
+    // [END dlp_deidentify_mask]
+  }
+
+  private static void deidentifyWithFpe(
+      String string, CryptoReplaceFfxFpeConfig.FfxCommonNativeAlphabet alphabet, String keyName, String wrappedKey) {
+    // [START dlp_deidentify_fpe]
+    // instantiate a client
+    try (DlpServiceClient dlpServiceClient = DlpServiceClient.create()) {
+
+      // The string to deidentify
+      // string = "My SSN is 372819127";
+
+      // The set of characters to replace sensitive ones with
+      // For more information, see https://cloud.google.com/dlp/docs/reference/rest/v2beta1/content/deidentify#FfxCommonNativeAlphabet
+      // alphabet = FfxCommonNativeAlphabet.ALPHA_NUMERIC;
+
+      // The name of the Cloud KMS key used to encrypt ('wrap') the AES-256 key
+      // keyName = "projects/YOUR_GCLOUD_PROJECT/locations/YOUR_LOCATION/keyRings/YOUR_KEYRING_NAME/cryptoKeys/YOUR_KEY_NAME";
+
+      // The encrypted ('wrapped') AES-256 key to use
+      // This key should be encrypted using the Cloud KMS key specified above
+      // const wrappedKey = "YOUR_ENCRYPTED_AES_256_KEY"
+
+      ContentItem contentItem =
+          ContentItem.newBuilder()
+              .setType("text/plain")
+              .setValue(string)
+              .build();
+
+      KmsWrappedCryptoKey kmsWrappedCryptoKey =
```

Comment:
Same comment as earlier, better comments around these objects and ideally some of these can get cleaned up on the surface. This looks very cumbersome to the user.